### ISSUE TITLE
NVDA doesn't read an adblock plus filter abonnement Adblock Warning Removal List

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4182]
**Reported by The_Dark_Man on 2014-06-09 17:07**
NVDA reads in the filter settings of Adblock Plus in Mozilla Firefoxthe filter abonnements in the list. I added the Adblock Warning Removal List. But it reads the other filter abonnements.

**Comment by Adriani90** [comment]
@The_Dark_Man is this still reproducible? If yes, it might be a General Firefox bug but I cannot reproduce this issue.

**Comment by Adriani90** [comment]
@ehollig I suggest to Close this issue since the author is not on Github anymore and the issue is very specific. I cannot reproduce it with NVDA 2018.4. If someone else reports it we can reopen it later. Thanks.

**Comment by ehollig** [comment]
Closing as invalid
